pkg/controller/infrastructure/actuator_test.go

```diff
@@ -54,6 +53,12 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/runtime/inject"
 )
 
+const (
+	terraformMainTFFilename      = "main.tf"
+	terraformVariablesTFFilename = "variables.tf"
+	terraformTFVarsFilename      = "terraform.tfvars"
+)
```

Comment:
You could use https://github.com/gardener/gardener/blob/master/extensions/pkg/terraformer/config.go#L32-L41

Reply:
Good catch, I had to search for the value, not for the variable name.